src/Training.cpp

```diff
@@ -263,9 +273,8 @@ void Training::dump_training(int winner_color, OutputChunker& outchunk) {
             out << plane[plane.size() - 1];
             out << std::dec << std::endl;
         }
-        // The side to move planes can be compactly encoded into a single
-        // bit, 0 = black to move.
-        out << (step.to_move == FastBoard::BLACK ? "0" : "1") << std::endl;
+        // The side to move planes are used to encode the komi
+        out << step.stm_komi << std::endl;
```

Comment:
@gcp Since this changes the format of the data to a float should the file format version be change to 3 (on line 308)?

Reply:
Good point. But is the binary format v2 actually in use other than as a temporary intermediate format? If not we may just change the definition/specification of v2... It's clear this format cannot support stm-for-komi.

Reply:
I don't think so? We never cycled the training format because that's such a mess during a run.

I've never used the v2 on-disk format, there hasn't been a point for me.

Reply:
Thanks for confirming.